contact-us.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Hi @xDAnkit , why u have added lang as en only

Reply:
Hi @sanskrati-1234 , That's a default value but we'll surely updating in the final version